### PR TITLE
added new parts list view

### PR DESCRIPTION
**Pull request by mkurek**
TODO: tests
zrzut ekranu 2015-03-11 o 14 40 46

### CODE REVIEW

**Comment by vi4m** [comment]
Please rename Search -> Search devices

File: src/ralph_assets/models.py [hunk 1]
```diff
@@ -489,6 +489,14 @@ def get_query(self, q, request):
             name__icontains=q).order_by('name')[:10]
 
 
+class PartModelLookup(RestrictedLookupChannel):
+    model = PartModel
+
```
Comment (xliiv) [inline src/ralph_assets/models.py]: maybe `min_length` should be set to 3?, or something..

**Comment by andrzej-jankowski** [comment]
:ok:

**Comment by xliiv** [comment]
:ok:

**Pull request merged**